nuxeo-ai-model/pom.xml

```diff
@@ -16,6 +16,16 @@
       <groupId>org.nuxeo.ai</groupId>
       <artifactId>nuxeo-ai-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ai</groupId>
+      <artifactId>insight-rest-client</artifactId>
+      <scope>compile</scope>
```

Comment:
as far as i remember compile is the default scope (so no need to reference it)

Reply:
ok, thanks